zerver/lib/integrations.py

```diff
@@ -132,6 +132,7 @@ def url_object(self):
         function='zerver.webhooks.github_webhook.view.api_github_webhook'
     ),
     WebhookIntegration('gitlab', display_name='GitLab'),
+    WebhookIntegration('gogs', display_name='Gogs'),
```

Comment:
if you want to have `display_name` with just first letter capitalized please omit the argument - it should be done automatically.

As you can see we use display_name only when we expect different display_name like `gosquared -> GoSquared` (two capitalized letters)

Sigh... `Greenhouse` is an exception ;-)